pitop/robotics/drive_controller.py

```diff
@@ -28,16 +28,19 @@ def __init__(self, left_motor_port="M3", right_motor_port="M0", name="drive"):
         self.right_motor_port = right_motor_port
         self.left_motor_port = left_motor_port
 
-        # TODO: increase accuracy of wheel_base and wheel_diameter with empirical testing
-        self._wheel_separation = 0.1725
+        self._wheel_separation = 0.1675
```

Comment:
Undocumented

Reply:
By undocumented what do you mean? I added it to the description, should I be documenting it elsewhere too?